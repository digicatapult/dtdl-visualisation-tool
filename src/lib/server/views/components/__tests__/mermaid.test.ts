import { expect } from 'chai'
import { describe, it } from 'mocha'
import MermaidTemplates from '../mermaid'

describe('HomepageTemplates', () => {
  it('should render homepage', async () => {
    const templates = new MermaidTemplates()
    const rendered = await templates.flowchart({ graph: '' })
    expect(rendered).to.matchSnapshot()
  })
})
