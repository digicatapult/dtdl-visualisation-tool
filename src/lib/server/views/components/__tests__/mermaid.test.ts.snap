// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomepageTemplates should render homepage 1`] = `"<!DOCTYPE html><html lang=\\"en\\"><head><script src=\\"/lib/mermaidConfig.js\\" type=\\"module\\"></script><link rel=\\"icon\\" type=\\"image/ico\\" sizes=\\"48x48\\" href=\\"/public/images/favicon.ico\\"/><link rel=\\"stylesheet\\" type=\\"text/css\\" href=\\"/public/styles/main.css\\"/><title>Mermaid - Homepage</title></head><body><pre class=\\"mermaid\\"></pre></body></html>"`;
